autoload/ale/handlers/go.vim

```diff
@@ -8,7 +8,7 @@
 " Description: moved to generic Golang file from govet
 
 function! ale#handlers#go#Handler(buffer, lines) abort
-    let l:pattern = '\v^([a-zA-Z]?:?[^:]+):(\d+):?(\d+)?:? ?(.+)$'
+    let l:pattern = '\v^v*e*t*:*\s*([a-zA-Z]?:?[^:]+):(\d+):?(\d+)?:? ?(.+)$'
```

Comment:
`*` repeats the previous character/expression 0 or more times, and there's no need to match on several of these characters. Try `%(vet:)?` for optionally matching on a non-capturing group.

Add tests with the new message format.

Reply:
I was looking into this but had a hard time trying to figure out how to test this. @w0rp do you know of any similar tests that I could look at?